backend/core/src/seeds/listings/listing-detroit-10157.ts

```diff
@@ -0,0 +1,182 @@
+}
+
+const oneBdrmUnit: UnitSeedType = {
+  numBedrooms: 1,
+  status: UnitStatus.occupied,
+}
+
+for (let i = 0; i < 20; i++) {
+  nccUnits.push(oneBdrmUnit)
+}
+
+const twoBdrmUnit = {
+  numBedrooms: 2,
+  status: UnitStatus.occupied,
+}
+
+for (let i = 0; i < 10; i++) {
+  nccUnits.push(twoBdrmUnit)
+}
+
+const nccListing: ListingSeedType = {
+  applicationDropOffAddress: null,
+  applicationFee: "25",
+  applicationMailingAddress: null,
+  countyCode: CountyCode.detroit,
+  costsNotIncluded:
+    "Water Included Resident Pays Electricity Resident Pays Gas Resident Pays Heat(Heat is gas.)",
+  CSVFormattingType: CSVFormattingType.basic,
+  disableUnitsAccordion: false,
+  displayWaitlistSize: false,
+  hrdId: "HRD10157",
+  leasingAgentPhone: "313-873-1022",
+  managementCompany: "KMG Prestige",
+  managementWebsite: "www.kmgprestige.com/communities/",
+  name: "New Center Commons",
+  status: ListingStatus.active,
+}
+
+export class Listing10157Seed extends ListingDefaultSeed {
+  async seed() {
+    const unitTypeStudio = await this.unitTypeRepository.findOneOrFail({ name: "studio" })
+    const unitTypeOneBdrm = await this.unitTypeRepository.findOneOrFail({ name: "oneBdrm" })
+    const unitTypeTwoBdrm = await this.unitTypeRepository.findOneOrFail({ name: "twoBdrm" })
+    const unitTypeThreeBdrm = await this.unitTypeRepository.findOneOrFail({ name: "threeBdrm" })
+    const unitTypeFourBdrm = await this.unitTypeRepository.findOneOrFail({ name: "fourBdrm" })
+
+    const property = await this.propertyRepository.save({
+      ...nccProperty,
+    })
+
+    const unitsToBeCreated: Array<Omit<UnitCreateDto, keyof BaseEntity>> = nccUnits.map((unit) => {
+      let unitType
+      switch (unit.numBedrooms) {
+        case 4:
+          unitType = unitTypeFourBdrm
+          break
+        case 3:
+          unitType = unitTypeThreeBdrm
+          break
+        case 2:
+          unitType = unitTypeTwoBdrm
+          break
+        case 1:
+          unitType = unitTypeOneBdrm
+          break
+        default:
+          unitType = unitTypeStudio
+      }
+      return {
+        ...unit,
+        unitType: unitType,
+        property: {
+          id: property.id,
+        },
+      }
+    })
+    await this.unitsRepository.save(unitsToBeCreated)
+    const applicationMethod: ApplicationMethod = await this.applicationMethodRepository.save({
+      type: ApplicationMethodType.ExternalLink,
+      acceptsPostmarkedApplications: false,
+      externalReference: nccListing.managementWebsite,
+    })
+
+    const assets: Array<AssetDtoSeedType> = [
+      {
+        label: "building",
+        fileId: "https://s3.amazonaws.com/photos.rentlinx.com/L800/4275662.jpg",
+      },
+    ]
+
+    const listingCreateDto: Omit<
+      DeepPartial<Listing>,
+      keyof BaseEntity | "urlSlug" | "showWaitlist"
+    > = {
+      ...nccListing,
+      applicationMethods: [applicationMethod],
+      assets: JSON.parse(JSON.stringify(assets)),
```

Comment:
The new way to associate a photo with a listing is through `listing.image` (https://github.com/CityOfDetroit/bloom/blob/5959ec49bab6e16d27c6508a421e701075e7a00f/backend/core/src/listings/entities/listing.entity.ts#L440). I'm not sure if everywhere is hooked up to consume from there (instead of `listing.assets`), though

Reply:
I'll leave it for now since the other seeds still use the assets field.